app/views/content/life-as-a-teacher/explore-subjects/maths/_article.html.erb

```diff
@@ -46,56 +37,35 @@
-    <ul>
-      <li>GCSEs at grade 4 (C) or above in English and maths</li>
-      <li>a bachelor's degree in any subject</li>
-    </ul>
-    <p>Maths teacher training courses have had successful applications from candidates with a range of degrees, including psychology, music, architecture and law.</p>
+     <%= render 'content/shared/qualifications-training/check_qualifcations' %>
+    <p><p><%= @front_matter ["subject"].titleize %> teacher training courses have had successful applications from candidates with a range of degrees, including psychology, music, architecture and law.</p>
     <p>Find out more about what <a href="/train-to-be-a-teacher/qualifications-you-need-to-teach">qualifications you need to train to teach</a>.</p>
-    <%= render Content::ExpanderComponent.new(
-    title: "check your qualifications",
-    text: "If you have qualifications from outside the UK, you'll need to show that they meet the standards set for teacher training in England.",
-    link_title: "You can get help comparing English and international qualifications.",
-    link_url: "/non-uk-teachers/non-uk-qualifications",
-  ) %>
+     <%= render 'content/shared/qualifications-training/non_uk_qualifications' %>
   </section>
 
   <section class="col col-space-m col-720">
-    <h3>Subject knowledge enhancement course (SKE)</h3>
-    <p>If your training provider thinks you need to top up your maths knowledge, you might need to do an <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">SKE course</a> before you start your teacher training.</p>
+    <%= render 'content/shared/qualifications-training/subject_knowledge' %>
   </section>
 
   <section class="col col-space-m col-720">
     <h3>Fund your teacher training</h3>
-    <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_maths %> or scholarships of <%= v :scholarships_maths %></a> are available for eligible trainee maths teachers.</p>
-    <p>You can get a bursary or scholarship alongside a tuition fee loan and maintenance loan.</p>
-    <p>You may also be able to get <a href="/funding-and-support">extra funding and support</a> if you're:</p>
-    <ul>
-      <li>a parent or carer</li>
-      <li>disabled</li>
-    </ul>
-    <%= render Content::ExpanderComponent.new(
-    title: "funding your training",
-    text: "If you do not have <a href=\"https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk\">indefinite leave to remain in the UK</a>, you're unlikely to be eligible for funding.",
-    link_title: "Find out more about fees and financial support for non-UK trainee teachers.",
-    link_url: "/non-uk-teachers/fees-and-funding-for-non-uk-trainees",
-  ) %>
+    <!-- Bursaries and Scholarships -->
+    <%= render 'content/shared/teaching/bursary_scholarship' %>
+    <!-- Funding and Support -->
+    <%= render 'content/shared/teaching/funding_support' %>
```

Comment:
I think this `funding_support` partial needs tweaking slightly @joegibb @Sarah-DfE 

In prod on the maths page, we say `You can get a bursary or scholarship alongside a **tuition fee loan and maintenance loan**.`

However, the partial says `You can get a bursary or scholarship alongside a **tuition fee and maintenance loan**.` This makes it sound like it is one loan, when in fact it is two different types of loan.

I think we need to go back to the original wording (or improve it!) - ie update the partial to make it clear that the tuition fee loan and maintenance loan are separate/different